src/index.js

```diff
@@ -45,3 +45,22 @@ exports.Queue = Plotly.Queue;
 
 // export d3 used in the bundle
 exports.d3 = require('d3');
+
+Plotly.register({
+    traces: {
+        bar: require('./traces/bar'),
```

Comment:
:+1:

I'll add a `moduleType` field too then. 

Do we want to get rid of the `_`'s though? I'm happy either way, so its up to you.